packages/schema/src/schema/buffer.json

```diff
@@ -0,0 +1,30 @@
+{
+  "type": "object",
+  "description": "Buffer",
+  "title": "IBuffer",
+  "required": ["InputLayer", "bufferDistance"],
+  "additionalProperties": false,
+  "properties": {
+    "InputLayer": {
+      "title": "InputLayer",
+      "type": "string",
+      "fcType": "App::PropertyLink",
+      "description": "Select an input layer for buffering"
+    },
+    "bufferDistance": {
+      "type": "number",
```

Comment:
We should be able to provide a default value here

Reply:
Do we also want `"multipleOf"` & `"minimum"`? Not sure

Reply:
Maybe a minimum of `0` makes sense